docs/UserGuide.md

```diff
@@ -75,168 +75,203 @@ Shows a message explaning how to access the help page.
+Format: `reset INDEX tut/TUTORIAL`
+
+- Resets the attendance of the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** (1, 2, 3, ...).
+- `TUTORIAL` can be in the format of:
+    - A positive number between 1 - 12 (inclusive) e.g. `1`.
+    - A list of numbers e.g. `[1,3,5]`.
+    - A range of two numbers e.g. `3-6`.
 
 Examples:
 
-- `find John` returns `john` and `John Doe`
-- `find an` returns `armin` (characters a and n are present in **a**rmi**n**) and `brian`
-- `find alex david` returns `Alex Yeoh`, `David Li`<br> 
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
-- `find t/ri` returns all contacts marked with tag `friends` and `isRich` etc. <br>
-- `find alex t/colleagues` returns `Alex Yeoh` and all contacts marked with tag `colleagues`<br>
-  ![result for 'find alex t/colleagues'](images/findAlexColleaguesResult.png)
+- `reset 1 tut/1` Resets the attendance of the 1st student in the contact list for tutorial 1.
+- `reset 1 tut/1-3` Resets the attendance of the 1st student in the contact list for tutorials 1 to 3.
+- `reset 1 tut/[2,4,12]` Resets the attendance of the 1st student in the contact list for tutorials 2, 4 and 12.
+
+Visual Effect:
+
+The specified tutorial box of the specified student will turn <span style="color:grey">grey</span>.
 
-### Sorting persons : `sort`
+|:exclamation: *To reduce visual clutter, an image will not be provided*.|
+---
 
-Sorts the displayed list of persons by either name, student id or tutorial attendance. The sorting order will be maintained while edits to the list are made.
+<div markdown="block" class="alert alert-info">
 
-Format: `sort ORDER [n/][i/][tut/TUTORIAL]`
+**:information_source: Faster attendance updating:**<br>
+You can use the `mark`, `unmark` and `reset` commands with the wildcard `*` to update the attendance of all currently displayed students in the contact list at once.
+- `mark * tut/1` marks all students as attended for tutorial 1.
+- `unmark * tut/1` marks all students as absent for tutorial 1.
+- `reset * tut/1` resets the attendance of all students for tutorial 1.
+</div>
+
+### Sorting students : `sort`
+
+Sorts the displayed list of students by either name, student id or tutorial attendance. The sorting order will be maintained while edits to the list are made.
+
+Format: `sort ORDER [n/] [i/] [tut/TUTORIAL]`
 
 - `ORDER` indicates whether the sorted list is **ascending** or **descending**.
 - **Ascending** is represented with integer 1.
 - **Descending** is represented with integer -1.
 - `n/` indicates sorting according to name.
 - `i/` indicates sorting according to student id.
 - `tut/TUTORIAL` indicates sorting according to tutorial attendance for a specific tutorial.
+- At least one sorting criterion `[n/]`, `[i/]`, `[tut/TUTORIAL]` must be present. `sort ORDER` is an invalid command.
```

Comment:
fix #208 
- state that at least one field must be present

Reply:
Should be "only one". We don't support multiple criterion.